src/analytics/redux/tracked_events.js

```diff
@@ -111,20 +196,10 @@ function handleBlockMovedByPosition( clientIds, toIndex ) {
 
 export const trackedEvents = {
 	'core/block-editor': {
-		insertBlock( blocks ) {
-			trackBlocksHandler(
-				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
-			);
-		},
-		insertBlocks( blocks ) {
-			trackBlocksHandler(
-				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
-			);
-		},
+		insertBlock: trackBlockInsertion,
+		insertBlocks: trackBlockInsertion,
+		replaceBlock: trackBlockReplacement,
```

Comment:
Same as the above comment, since `replaceBlock` is invoking `replaceBlocks`, could we simply track events for the latter?

https://github.com/WordPress/gutenberg/blob/d27c43fda419995a80b1cbdeafe2b24d9a0e2164/packages/block-editor/src/store/actions.js#L416-L418

Reply:
More context of this approach in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976#discussion_r1268144468 but after some testing I realized that only the action that was dispatched initially is the one tracked